src/main/java/com/synopsys/integration/alert/web/actions/DescriptorMetadataActions.java

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
This class contains the contents of the DescriptorController.  This provides methods to create the descriptor metadata to be used in both the about page and the user based descriptor data.